### PR TITLE
VONK-12: describe X-Provenance header use

### PR DESCRIPTION
**Pull request by DavidKloet**

### CODE REVIEW

File: features/x-provenance.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+X-Provenance header
+===================
+
+Description
+-----------
+
+The X-Provenance header can be used to add a Provenance resource upon creating or updating another resource. See `Provenance#header <https://www.hl7.org/fhir/Provenance.html#header>`_ for more information.
+
+.. note:: In the case of a conditional create, where the resource was already present, the Provenance resource is not created.
```
Comment (alexzautke) [inline features/x-provenance.rst:11]: Probably also conditional update?

**Review by alexzautke — APPROVED** [review]
Approved with question

File: features/x-provenance.rst (hunk 1 above)
Comment (DavidKloet) [reply features/x-provenance.rst:11]: That's actually a question I have. According to the code and https://www.hl7.org/fhir/http.html#cond-update we always do something, correct? I do know there's a 'If-Match: [etag]' or soemthing, but I can't find it in the code

Comment (alexzautke) [reply features/x-provenance.rst:11]: You are right, in all cases we either return an error or create a new resource. We implement the if-match header but if that doesn’t match than we return an 4xx status code

**Pull request merged**